maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js

```diff
@@ -5,11 +5,15 @@ import "bootstrap/dist/css/bootstrap.css";
 import UserSearch from "./UserSearch";
 
 const userSearch = {
-  state: { isLoading: false }
+  state: { isLoading: false, userExists: true }
 };
 
 const isLoadingUserSearch = {
-  state: { isLoading: true }
+  state: { isLoading: true, userExists: true }
+};
+
+const userNotExist = {
```

Comment:
call this `noUserExists`

Reply:
we dont need an entire object for this... this should be cleaned up in terms of naming